README.md
Kimai-i Time Tracking v0.10
====================================
(originates from Kimai v0.9.3rc1)

Some of the features:

    - Implementation of mysqli (and eliminition of mysql-php-4)
    - Tested on PHP 5.6 (demo runs on PHP 5.5)
    - Revision of rendering of Timesheet, Export, Expenses, Invoice (header & content alignments)
    - New skin, better GUI, more intuitive
    - Code revision & optimisation... now closer to PSR1.
    - Many bugfix & enhancements
    - "root" mode, overrides all 'user roles' (great for single and few people team)
    - Automatic database upgrade from previous version of Kimai (up to v0.9.3)
    - ...

Try it!

Claude.



kimaii.demo.ws01.info
=====================
    - Demo in Deutsch, English, Español, Français (Deutch & Español ~ partially translated)
    - Login as: admin, staff1 or customer1
    - Password: demo
    - Data resets every 6 hours

Your feedback is welcome.



For developpers and enthousiasts
================================
    - The code has been revised to about 70% (appart from third parties libraries).
    - Better coherence in the structure, naming convention and style (PSR1-2 as reference)
    - Deprecated code has been eliminated. Big clean up done.
    - Focused more on simplicity & practicality & stability. 



What you can do here
====================
    - Download it
    - Ask questions
    - Post bug & solution you find





core/CHANGES.TXT
*************** REMINDER ... UPDATE /includes/version.php ******************************

=== v0.10.1-b2 ======= 2015-08-06 ============ db revision 2000 ========================================================
- bugfix: time of day difference between client & server: Messes up with the time recording. Server was behind and
    timer would start at -50 second because of a 10 second difference. I removed this attempt of managing the offset
    between client & server, because in fact it is irrelevant. What is, is to have precise delta between start and
    stop. Since it is now ONLY the server's that is used, for both start/end. So even if client is 5 minutes behind,
    the user may see a difference in the hour:minutes, it will show in the start as well as the end, but the gap is
    valid.
- some corrections in query results evaluations.



=== v0.10.1-b1 ======= 2015-08-06 ============ db revision 2000 ========================================================
- bugfix: invoice panel width not following. Solved.
- demo: some attempt to figure out how ki_demo (demoextention) is supposed to perform, or if it even ever worked.(!)
- DEMO_MODE: introduction of DEMO_MODE to enable/disable some feature.
    Nothing to do with ki_demo(demoextension).
- centralized cookie creation
- language: now selectable on login page. Can also be set in the uri (?language=en), practical when having different
    language customers, and need to send them a link.
- revision of language selection process when initializing. Inclusion of cookie ki_language. Priority: uri, cookie,
    pref, default and english if none is available.  Also, language decision is centralized in transaction.class,
    nowhere else. Simplicity please!



=== v0.10.0-b5 ======= 2015-08-04 ============ db revision 2000 ========================================================
- start and stop recording time are now both taken from the server.  And it will show in local time as define in the
    user's preferences.
- minor corrections
- correction in delete_user




=== v0.10.0-b4 ======= 2015-07-27 ============ db revision 2000 ========================================================
- strengthen even more the choosing of update/install process with cleaner/safer redirection.

- revised internal version check. Now, if no db revision, then only update core.version number. No need to run updater.



=== v0.10.0-b3 ======= 2015-07-27 ============ db revision 2000 ========================================================
- grrr. having some difficulties with git...



=== v0.10.0-b2 ======= 2015-07-27 ============ db revision 2000 ========================================================
- languages: some modifications in EN and FR.
- bugfix: iconv_set_encoding deprecated WARNINGS with PHP v5.6: Made new function ki_iconv_set_encoding to replace
    iconv_set_encoding. Function checks PHP version. This affects only zend framework 1.
- bugfix: java validation of time in timesheet edit. We do not care about seconds!!



=== v0.10.0-b1 ======= 2015-07-25  ============ db revision 2000 =======================================================
Notes: This jump to v0.10.0 db revision 2000, is my major overhaul of original Kimia v0.9.3rc1. There is still work to
do, but from what I have revised, the core is functionnal.  Here a short and incomplete list of what I have done.

- improvement: removed all references to 'server_conn', and 'pdo', as there are NO pdo processing in Kimai.
- improvement: removed 'server_type', unused.
- improvement: removed abstract mysql class, simply unnecessary with no real gain.
- improvement: added all default preferences for new users in global config.
- improvement: removed one unnecessary database locks ( no need when not multi table updates, even less when only select)
- improvement: upgraded the db transaction handling method
- feature: preferences is now unified, for customer or user. Methods, Variables & db. User/Customer's prefs are both in
    the preference table.
- improvement: user_id & customer_id are now both checked for uniqness against preference table.
- features: customer can now set their own preferences.
- improvement: massive revision of global variables (some people should learn about why and how to use
    global/protected/private variables/method, and abstract)
    ~ GLOBALS: kga, database, view (zend framework), translation.
    ~ important $kga arrays:
        * any       (id, type, common to user & customer)
        * conf      (copy of configuration table)
        * customer  (when logged-in, customer's data)
        * lang      (locale translation)
        * pref      (preferences of both user/customer)
        * status    (copy of status table)
        * user      (when logged-in, user's data)
- improvement: handling of grid header in timesheet,expenses,export...
- improvement: PSR standardizing of all namings of tables and rows (excluding role related)
- improvement: PSR standardizing of "role" related row names in db-tables. no more "-", instead "_".
- improvement: status table now loaded in translation array, what ever translation exist will
    override status name from db... Simply put, change language and status names
    will follow. If no translation exist (because it's new status) then the status
    name from the db will be shown.
- improvement: debug panel: deactivate log updates while panel not active (avoid useless http requests)
- feature: extention's css & grfx folders can now be included in a skin folder, then modified to change the appearance

core/language/cz.php
<?php
/**
 * This file is part of
 * Kimai - Open Source Time Tracking // http://www.kimai.org
 * (c) 2006-2009 Kimai-Development-Team
 *
 * Kimai is free software; you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation; Version 3, 29 June 2007
 *
 * Kimai is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with Kimai; If not, see <http://www.gnu.org/licenses/>.
 */
 
/**
 * czech
 *
 */
return array(
"countryCode"       => "cz-CZ",
"preferences"       => "Předvolby",
"total"		        => "Celkem",
"monatzeit"         => "Měsíc",
"heute"             => "dnes",
"gestern"           => "včera",
"autoOn"            => "Auto On",
"autoOff"           => "Auto Off",
"datum"             => "Datum",
"in"                => "Od",
"out"               => "Do",
"time"              => "h'm",
"timelabel"         => "Čas",
"durationlabel"     => "Doba",
"customer"          => "Zákazník",
"project"           => "Projekt",
"activity"          => "Task",
"customers"         => "Zákazníci",
"projects"          => "Projekty",
"activities"        => "Tasky",
"admin"             => "Admin",
"rowlimit"          => "max. řádků",
"breiteZeitspalte"  => "width time column",
"breiteUhrspalte"   => "width clock column",
"breiteDatumspalte" => "width date column",
"recommended"       => "doporučeno",
"submit"            => "OK",
"cancel"            => "Cancel",
"edit"              => "Edit",
"construction"      => "v přípravě...",
"about"             => "O Kimai",
"hours"             => "hodin",
"mins"              => "minut",
"secs"              => "sekund",
"skin"              => "Vzhled",
"erase"             => "Smazat",
"new_customer"      => "přidat zákazníka",
"new_project"       => "přidat projekt",
"new_activity"      => "přidej task",
"day_summary"       => "Časový přehled",
"plzChoose"         => "prosím vyber",
"noEntries"         => "Nejsou trekovatelná data pro specifikovaný čas...",
"recordAgain"       => "Zaznamenej znovu",
"comment"           => "Komentář",
"location"         => "Umístění",
"trackingNumber"        => "Tracking Number",
"cookiewarning"     => "Před použitím zapni cookies.",
"JSwarning"         => "JavaScript musí být zapnutý.",
"banned"            => "Banned (Účet zablokován)",
"tooManyLogins"     => "Mnoho špatných přihlášení - user suspended.",
"accessDenied"      => "Přístup zakázán!",
"wrongPass"         => "Špatný uživatel a/nebo heslo",
"tryAgain"          => "Zkus znovu / zapoměl jsi heslo?",
"back"              => "Zpět na přihlášení",
"bye"               => "Nashle!",
"login"             => "Login",
"logout"            => "Logout",
"loggedon"          => "Přihlášen jako:",
"username"          => "Uživatelské jméno",
"password"          => "Heslo",
"newPassword"       => "Nové heslo",
"retypePassword"    => "Znovu heslo",
"passwordsDontMatch"=> "Hesla se neshodují.",
"passwordTooShort"  => "Heslo nemá nejméně 5 znaků.",
"minLength"         => "min. 5 znaků!",
"update"            => "Struktura databáze musí být aktualizována.<br />Doporučujeme pro jistotu zálohovat záznamy<br />aby nedošlo k jejich ztrátě v průběhu aktualizace.",
"updateNow"         => "Aktualizuj databázi!",
"stop"              => "Stop!",
"start"             => "Start!",
"setTimeframe"      => "set timeframe",   
"now"               => "Teď",
"specify"           => "specify list",
"add"               => "Přidej",
"running"           => "Běží...",
"company"           => "Společnost",


"vat"               => "DPH",
"contactPerson"     => "Kontaktní osoba",


"street"            => "Ulice",
"zipcode"           => "PSČ",
"city"              => "Město",
"telephon"          => "Telefon",
"fax"               => "Fax",
"mobilephone"       => "Mobil",
"mail"              => "E-Mail",
"homepage"          => "Internet",
"users"             => "Uživatel",
"name"              => "Jméno",
"alias"             => "Přezdívka",
"adduser"           => "Přidat uživatele",
"addgroup"          => "Přidat skupinu",
"group"             => "Skupina",
"groups"            => "Skupiny",
"advanced"          => "Rozšířené",
"database"          => "Databáze",
"options"           => "Možnosti",
"status"            => "Stav",
"adminmail"         => "email Administrátora",
"logintries"        => "Počet chybných přihlášení před zablokováním",
"bantime"           => "čas blokování přístupu",
"lastdbbackup"      => "Poslední záloha",
"runbackup"         => "Zálohuj teď",
"defaultGroup"        => "Default (0)",
"user"              => "Regular user",
"editUser"          => "Editace uživatele",
"editGroup"           => "Editace skupiny",
"delete_group"            => "smazání skupiny",
"delete_project"    => "smazání projektu",
"delete_customer"   => "smazání zákazníka",
"delete_activity"   => "smazání tasku",
"switchUser"        => "Přepnout na tohoto uživatele",
"backupUser"        => "Zálohuj tohoto uživatele",
"mailUser"          => "Pošli email tomuto uživateli",
"deleteUser"        => "Smaž tohoto uživatele",
"bannedUser"        => "Blokovaný uživatel - klikni k reaktivaci",
"activeAccount"     => "Aktivní uýivatel - klikni k blokování",
"members"           => "Členové",
"nopassword"        => "Nenastavené heslo!",
"noItems"           => "žádné údaje",
"DBname"            => "Jméno databáze",
"autoselection"     => "Automatický výběr",
"quickdelete"       => "Option to delete entries from within tables",
"quickdeleteHide"   => "Ne",
"quickdeleteShow"   => "Ano, bez potvrzovacího dialogu",
"quickdeleteShowConfirm" => "Ano, s potvrzovacím dialogem",
"tools"             => "Nástroje",
"print"             => "Tisk",
"save"              => "Ulož",
"export"            => "Export",
"groupname"         => "Jméno skupiny",
"help"              => "Help",
"select"            => "Vyber",
"hidedeletedusers"  => "Skryj smazané uživatele",
"showdeletedusers"  => "Ukaž smazané uživatele",


"month"             => "Měsíc",
"day"               => "Den",
"year"              => "Rok",


"lang"              => "Jazyk",


"ac"                => "Automatically continued entry.",
"ac_error"          => "Automatically continued entry with error (timespan too long).",
"refundable_long"   => "Refundable (to employee)",
"refundable"        => "Refundable",
"yes"               => "Ano",
"no"                => "ne",
"commentType"      => "Typ komentáře",
"ctype0"            => "Komentář",
"ctype1"            => "Oznámení",
"ctype2"            => "Upozornění!",
"hint"              => "Nápověda:",
"rename_caution_before_username"      => "pokud změníte uživatelské jméno",
"rename_caution_after_username"      => "Musíte se přihlásit ihned znovu pod novým jménem!",
"timediff_warn"     => "Musíte vložit rozdílné hodnoty do položek od-do!",
"visibility"        => "Viditelnost",
"showitem"          => "zviditelni",
"hideitem"          => "zneviditelni",
"checkUsername"     => "You must enter a username in the field left from the add-button!",
"checkGroupname"    => "You must enter a group name in the field left from the add-button!",
"clickToReload"     => "Klikni pro obnovení GUI",
"sure"              => "Opravdu?",
"clock"             => "Hodiny",
"currSum"           => "Current Timeframe Summary",
"admWarn"           => "You are Admin. Status can only be set lower by another admin.",
"flip_project_display"  => "Order project table by customers",
"project_comment_flag"  => "Always show comments beside projects",
"showIDs"           => "Ukaž IDčka",
"error"             => "Nastala chyba!",
"checkupdate"       => "zkontroluj aktualizace",


"selectedForRecording"           => "Vybraný záznam:",
"selectedCustomerLabel"         => "Z:",
"selectedProjectLabel"         => "P:",
"selectedActivityLabel"         => "T:",


"testCustomer"      => "Test-Customer",
"testProject"       => "Test-Project",
"testActivity"      => "testing",


"budget"            => "Rozpočet",
"rate"              => "Sazba (za hodinu)",
"rate_short"        => "Sazba",
"default_rate"      => "Základní sazba",
"my_rate"           => "Moje sazba",


"wage"              => "Hodnota",


"customerlogin"     => "zákaznický login",


"expense"           => "Náklad",
"designation"       => "Označení",


"cleared"           => "cleared",


"atLeastOneGroup" => 'Alespoň jedna skupina musí být vybrána.',


"noFading"          => "no fading effect",
"user_list_hidden"  => "hide user list by default",


'show_sensible_data' => 'Zobraz citlivá data na záložce debug.',
'show_update_warn' => 'Vypni varování před aktualizací.',
'check_at_startup' => 'Zobraz zprávu o nových verzích na přihlašovací stránce.',
'show_daySeperatorLines' => 'Nakresli čáru mezi položka z různých dní.',
'show_gabBreaks' => 'Nakrasli čáru mezi položkami které přímo nenavazují.',
'show_RecordAgain' => 'Zobraz tlačítko &quot;Zaznamenej znovu&quot;přek každým záznamem eventu.',
'show_TrackingNr' => 'umožni měnit Tracking-Nr.',
'currency_name' => 'Název měny.',
'currency_sign' => 'Označení použité měny.',
'currency_first' => 'Zobraz měnu před částkou (e.g. &euro; 100,00)',
'display_date_format' => 'Formát data v časovém rozmezí (<a href="http://php.net/manual/en/function.strftime.php" target="_blank">notation for strftime()</a>)',
'display_currentDate_format' => 'Formát data pod časovým rozmezím (<a href="http://php.net/manual/en/function.strftime.php" target="_blank">notation for strftime()</a>)',
'table_date_format' => 'Formát data v tabulkách (<a href="http://php.net/manual/en/function.strftime.php" target="_blank">notation for strftime()</a>)',
'round_time' => 'Zaokrouhli čas na ',
'round_time_minute' => ' minute(s).',
'logged_in_as' => 'přihlášen jako:',


'decimal_separator'  => 'Oddělovač desetin',


'view_filter' => 'filtruj',
'unassigned' => 'nepřiřazené',
'all_activities' => '&lt;&lt;vše&gt;&gt;',


'durationWithSeconds' => 'Ukaž trvání s přesností na sekundy.',


'multiplier' => 'Multiplier',


'dateAndTimeHelp' => '<strong>Časy a data nejsou zadány v následujícím formátu:</strong><br />
            Data: DD.MM.YYYY<br />
            Časy: HH:MM:SS nebo HH:MM nebo jen HH. vynechané části budou nastaveny na 0.',


'timezone' => 'časová zóna',
'defaultTimezone' => 'Časová zona pro nové zákazníky a uživatele',


'hideClearedEntries' => 'hide cleared entries',


'general' => 'Hlavní',
'address' => 'Adresa',
'contact' => 'Kontakt',

"filter" => "Filtr",

'internalProject' => 'interní projekt',


'exactSums' => 'Use exact sums instead of adding the shown rounded values.',


'securityWarning' => 'bezpečnostní varování',
'installerWarningHeadline' => 'instalační adresář stále existuje!',
'installerWarningText' => 'Smaž instalační adresář aby někdo jiný nemohl provést instalaci.',


'change' => 'změň',


'sublistAnnotations' => 'sublist annotations',


'TimeDateInputError' => 'Datum nebo čas není platný.',


'StartTimeBeforeEndTime' => 'Čas začátku misí být před koncem.',




'credits' => <<<EOF
        Torsten H&ouml;ltge, Severin Leonhardt, Kevin Papst, Oleg Britvin, Martin Klemkow and others...<br />
        
        <br />
        
        <strong>This program is free software and published 'AS IS' without any kind of warranty</strong> under the terms of the
        <a href="../COPYING">GPL Version 3</a>. 
        
        <br /><br />
         
        <a href='http://www.kimai.org' target='_blank'>Official Website</a> | 
        <a href='http://forum.kimai.org/' target='_blank'>Forum</a> |
        <a href='https://github.com/kimai/kimai' target='_blank'>GitHub</a> |
        <a href='http://sourceforge.net/projects/kimai/' target='_blank'>Sourceforge</a> |
        <a href='http://www.kimai.org/en/download.html' target='_blank'>Download</a>
        
        <br /><br />


        If you do any alternations we would appreciate a posting on the <a href="http://forum.kimai.org" title="Kimai Forum">forum</a>.
        You'll find a special board "<a href="http://forum.kimai.org/index.php?board=10.0" title="Extensions / Friendly Hacks">Friendly Hacks</a>" there.
        Latest Developer Version can be downloaded <a href="https://github.com/kimai/kimai/archive/master.zip">here</a> (download for testing purposes only!).
        
        <br /><br />
        
        Please support further development of Kimai with a <a href="http://www.kimai.org/en/donate.html">donation</a> or by
        extending its <a href="https://github.com/kimai/kimai/tree/master/core/language">language library</a>.


        If you find bugs or have ideas for features, feel free to contact us through the forum.
        Or even better: create a <a href="https://github.com/kimai/kimai/issues">ticket</a>!<br /><br />


        <strong>Further thanks for support to:</strong> Vasilis van Gemert, Maximilian Kern, Enrico Ties, Thomas Wensing, John Resig, Kelvin Luck, Urs Gerig, Willem van Gemert,
        Torben Boe and HamBug Studios, Klaus Franken, Chris (Urban Willi), Andreas Berndt, Niels Hoffmann, G&uuml;nter Hengsbach, Paul Brand, Joaqu&iacute;n G. de la Zerda,
        Allesandro Bertoldo, Jos&eacute; Ricardo Cardoso.<br /><br />


        <strong>Special thanks</strong> to all beta testers, which are way to many to mention them all - you know who you are! ;)<br /><br />


        The invoice extension uses the PclZip Library, which can be found at <a href="http://www.phpconcept.net" rel="nofollow">http://www.phpconcept.net</a>.<br/><br/>


        Version 0.8 was emerged with the kind support of RRZE (Regionales Rechenzentrum Erlangen).<br /><br />
        <br />
EOF
,




'months'         => array("Leden","Únor","Březen","Duben","Květen","Červen","Červenec","Srpen","Září","Říjen","Listopad","Prosinec"),
'months_short'   => array("Led","Únr","Bře","Dub","Kvě","Čer","Čec","Srp","Zář","Říj","Lis","Pro"),
'weekdays'       => array("Neděle","Pondělí","Úterý","Středa","Čtvrtek","Pátek","Sobota"),
'weekdays_short' => array("Ne","Po","Út","St","Čt","Pá","So"),


'errors' => array (
  0 => array ('hdl' => "Nemohu se připojit k databázi!",
              'txt' => "Zkontroluj zda databáze běží, hje aktivní PDO a údaje v 'autoconf.php' v adresáři 'includes' jsou v pořádku."),


  1 => array ('hdl' => "Chyba v průběhu instalace!",
              'txt' => 'Kimai is unable to create necessary tables because tables with the name name already exist.<br/>
To update your existing installation please click <a href="../index.php">here</a>.<br/>
Choose another table prefix or delete existing tables with this prefix in order to allow Kimai to perform a clean installation.')
  ),


'updater' => array (
0        => "Yes, I have a backup of my Kimai database! Proceed updating!",
10       => "Legend:",
20       => "Query was executed successfully.",
30       => "Query caused error - most likely irrelevant.",
40       => "Query caused fatal error.",
50       => "Backup Log:",
60       => "Backup failed - Update aborded!",
70       => "Update Log:",
80       => "The database was updated successfully and a backup was created.",
90       => "Queries processed.",
100      => "The database-update failed due to fatal error(s). A backup was made which can be used to recreate the database in case Kimai does not work properly.",
110      => "New Passwords",
120      => "See below for new passwords for every user.",
130      => "Could not write to autoconf.php.",
140      => "update file autoconf.php",
'timezone' => 'Kimai now supports timezones. Please select your default time zone. It will be used for all users and customers. Users can change their time zone.',
),


'backup' => array(
0        => "This backup restore utility is not yet tested! Use at your own risk!",
1        => "Dostupné zálohy:",
2        => "obnov",
3        => "smaž",
4        => "OK",
5        => "You can only recover *one* backup at a time!",
6        => "Backup from",
7        => "was recovered.",
8        => "vytvoř zálohu",
9        => "These backups are actually snapshots! Meaning they still live within the very same database as your actual Kimai work data. Backups via this utility are NO substitution for real backups on a physically different data medium. There are plenty of tools for database backups available on the web ;)"),



'export_extension' => array(
	
"print_hint"      		   => "Click OK to open a printable table.",
"dl_hint"         		   => "Click OK to download export-file.",
                              
"tab_filter"	 		   => "Filtr",
"tab_location"	 		   => "Umístění",
"tab_timeformat"	 	   => "Formát času",
                              
"cleared_all"	 		   => "vše",
"cleared_cleared" 		   => "only cleared",
"cleared_open"	 		   => "only open",
"stdrd_location"  		   => "Standard-Location",
"timeformat"	     	   => "Formát času",
"dateformat"			   => "Formát data",

"refundable_all"	 	   => "vše",
"refundable_refundable"    => "only refundable",
"refundable_not_refundable"=> "only non refundable",
                              
"print_comment"   		   => "tiskni komentáře",
"print_summary"   		   => "tiskni sumář",
"create_bookmarks"		   => "create bookmarks",
"download_pdf"    		   => "stáhni jako PDF",
"pdf_format"               => "formát",
"export_pdf"               => "tabulka",
"export_pdf2"              => "seznam",
                              
"column_delimiter"		   => "delimiter for columns",
"quote_char"      		   => "boundary character for fields",
                              
"exportCSV"         	   => "CSV Export",
"exportPDF"         	   => "PDF Export",
"exportXLS"         	   => "Excel Export",
"print"         	       => "Tisk",
"export_timeformat_help"   => "Nápověda formátu času",


'subtotal'              => 'Mezisoučet',
'finalamount'           => 'Celkem',
'pdf_headline'          => 'Seznam odpracovaných časů a nákladů',
'pdf_headline_only_times'   => 'Seznam odpracovaných časů',
'pdf_headline_only_expenses'=> 'Seznam nákladů',
'costs'                 => 'Costs',
'expense'               => 'Náklady',
'summary'               => 'Sumář',
'full_list'             => 'Úplný výpis',
'page'                  => 'strana',
'by'                    => 'od',
'done_by'               => 'done by',
'duration'              => 'Trvání',
'customer_new_page'     => 'nová strana pro každého zákazníka',


'times_and_expenses' => 'časy a náklady',
'times' => 'časy',
'expenses' => 'náklady',


'reverse_order' => 'starší nejdřiv',
'time_period'  => 'Období',


'duration_unit' => 'h',


),




'ext_invoice' => array(
	
"invoiceTitle" 			=> "Tvorba faktury",
"invoiceProject" 		=> "Projekt:",
"invoiceTimePeriod" 	=> "Období:",
"invoiceOptionShort" 	=> "Krátká faktura",
"invoiceOptionRound" 	=> "Zaokrouhlení času",
"invoiceButton" 		=> "Vytvoř",
'noData'                => 'Nejsou data pro zvolený projekt a období.',
'defaultVat'        => 'defaultní DPH'
),


'ext_budget' => array(
  'unusedBudget' => 'unused'
),


'extensions' => array(
'ki_timesheet' => 'Časy',
'ki_expenses' => 'Náklady',
'ki_invoice' => 'Faktury',
'adminPanel_extension' => 'Administrace',
'ki_budget' => 'Rozpočet'
),

"log_delete"	=> "přihlásit vymazána",

);



?>
